multigeiger/multigeiger.ino

```diff
@@ -65,10 +66,19 @@ void setup() {
   setup_display(isLoraBoard);
   setup_switches(isLoraBoard);
   switches = read_switches();  // only read DIP switches once at boot time
+  if (!sendToBle)
+    switches.ble_on = false; // avoid switching on in WebConfig and trying to use BLE without setup
   setup_thp_sensor();
   setup_webconf(isLoraBoard);
   setup_speaker(playSound, ledTick && switches.led_on, speakerTick && switches.speaker_on);
   setup_transmission(VERSION_STR, ssid, isLoraBoard);
+  // BT BLE
+  if (sendToBle && switches.ble_on) {
+    set_status(STATUS_BT, ST_BT_INIT);
+    setup_ble(ssid);
+  } else {
+    set_status(STATUS_BT, ST_BT_OFF);
+  }
```

Comment:
just move this code **into** setup_ble() so it is just calling that from here?

Reply:
I wanted to avoid mixing too many modules, but I guess a status call from Lora and BLE is completely understandable. Will move it there.

Reply:
done